[debug] [2019-02-01T20:24:55.727Z] ----------------------------------------------------------------------
[debug] [2019-02-01T20:24:55.729Z] Command:       /usr/local/bin/node /usr/local/bin/firebase serve
[debug] [2019-02-01T20:24:55.729Z] CLI Version:   6.3.1
[debug] [2019-02-01T20:24:55.729Z] Platform:      darwin
[debug] [2019-02-01T20:24:55.729Z] Node Version:  v8.11.3
[debug] [2019-02-01T20:24:55.729Z] Time:          Fri Feb 01 2019 21:24:55 GMT+0100 (CET)
[debug] [2019-02-01T20:24:55.729Z] ----------------------------------------------------------------------
[debug] 
[debug] [2019-02-01T20:24:55.738Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2019-02-01T20:24:55.738Z] > authorizing via signed-in user
[debug] [2019-02-01T20:24:55.738Z] [iam] checking project autumn-md for permissions ["firebase.projects.get"]
[debug] [2019-02-01T20:24:55.739Z] >>> HTTP REQUEST POST https://cloudresourcemanager.googleapis.com/v1/projects/autumn-md:testIamPermissions  

[debug] [2019-02-01T20:24:56.746Z] <<< HTTP RESPONSE 200
[debug] [2019-02-01T20:24:56.747Z] >>> HTTP REQUEST GET https://cloudresourcemanager.googleapis.com/v1/projects/autumn-md  
 
[debug] [2019-02-01T20:24:59.697Z] <<< HTTP RESPONSE 200
[info] 
[info] === Serving from '/Users/mrdavid/autum_5'...
[info] 
[debug] [2019-02-01T20:25:00.581Z] Error: EACCES: permission denied, mkdir '/usr/local/lib/node_modules/firebase-tools/node_modules/@google-cloud/functions-emulator/logs'
    at Object.fs.mkdirSync (fs.js:885:18)
    at Object.assertLogsPath (/usr/local/lib/node_modules/firebase-tools/node_modules/@google-cloud/functions-emulator/src/emulator/logs.js:65:10)
    at new Controller (/usr/local/lib/node_modules/firebase-tools/node_modules/@google-cloud/functions-emulator/src/cli/controller.js:77:32)
    at FunctionsEmulator.start (/usr/local/lib/node_modules/firebase-tools/lib/functionsEmulator.js:102:23)
    at Object.start (/usr/local/lib/node_modules/firebase-tools/lib/serve/functions.js:7:33)
    at /usr/local/lib/node_modules/firebase-tools/lib/serve/index.js:15:23
    at arrayMap (/usr/local/lib/node_modules/firebase-tools/node_modules/lodash/lodash.js:639:23)
    at Function.map (/usr/local/lib/node_modules/firebase-tools/node_modules/lodash/lodash.js:9556:14)
    at _serve (/usr/local/lib/node_modules/firebase-tools/lib/serve/index.js:13:26)
    at Command.module.exports.Command.description.option.option.option.option.before.action [as _action] (/usr/local/lib/node_modules/firebase-tools/lib/commands/serve.js:55:12)
[error] 
[error] Error: An unexpected error has occurred.
